protocols/mdns/Cargo.toml

```diff
@@ -46,3 +46,7 @@ required-features = ["async-io"]
 name = "use-tokio"
 required-features = ["tokio"]
 
+[package.metadata.docs.rs]
+all-features = true
+rustdoc-args = ["--cfg", "docsrs"]
+rustc-args = ["--cfg", "docsrs"]
```

Comment:
I think it may be worthwhile to just copy-paste this into all manifests? Otherwise, if we add features later we may forget about this.

We can include a link to [this](https://docs.rs/about/builds#cross-compiling) documentation to make this a bit easier to understand.